lua/nvim-tree/view.lua

```diff
@@ -201,12 +201,31 @@ function M.close()
       if vim.api.nvim_win_is_valid(tree_win) then
         vim.api.nvim_win_close(tree_win, true)
       end
+      if all_tabpages then
+        for _, v in pairs(M.View.tabpages) do
+          if v.winnr and vim.api.nvim_win_is_valid(v.winnr) then
+            vim.api.nvim_win_close(v.winnr, true)
+          end
+        end
+      end
```

Comment:
This is failing when the tree is not open in the current tab:

```lua
    tab = {
      sync = {
        open = false,
        close = false,
        ignore = {},
      },
    },
```

Fail:
```
:NvimTreeOpen
:tabnew
:lua require"nvim-tree.api".tree.close_in_all_tabs()
```

Pass:
```
:NvimTreeOpen
:tabnew
:NvimTreeOpen
:lua require"nvim-tree.api".tree.close_in_all_tabs()
```

It looks like we will need to move the all-tabs-closing bit out of the nvim_list_wins loop.